src/mobile/apps/artwork/components/image/templates/index.jade

```diff
@@ -21,7 +21,6 @@
         else if artwork.cultural_maker
           .artist-name= artwork.cultural_maker
         
-        unless artwork.auction || (artwork.auction && artwork.auction.is_auction)
-          include ./details.jade
+        include ./details.jade
```

Comment:
👍 looks good to me.

Reply:
This looks legit to me as well. I'm merging this so we can deploy.